### DISCUSSION TITLE
Does Bun v1.2.25 node:cluster now work with pm2 in cluster mode?

### DISCUSSION THREAD

**Discussion by dienstbereit** [oven-sh/bun#13546] [General]
Is it possible to start bun with pm2 in cluster mode after the integration of node:cluster support in bun 1.2.25? And if so, how? I can't get it to work :(

**Comment by rgillan** [comment]
Was a while since we used pm2/cluster mode but doesn't pm2 itself run the clustering process (not the application you are clustering)?  From our experience a few years back, you either implemented the cluster yourselve (nodejs) or pm2 did it for you (not sure you can use pm2 cluster mode on an app itself trying to cluster).

**Reply by dienstbereit** [reply]
I assume that pm2 executes the ‘cluster mode’. Unfortunately, this does not seem to work with bun out-of-the-box. I would like to know why this is the case. Is it pm2 or bun? Maybe someone has an answer :)
